polaris-core/src/main/java/org/apache/polaris/core/PolarisConfiguration.java

```diff
@@ -154,6 +154,14 @@ public static <T> Builder<T> builder() {
           .defaultValue(false)
           .build();
 
+  public static final PolarisConfiguration<Boolean> ALLOW_EXTERNAL_CATALOG_CREDENTIAL_VENDING =
+      PolarisConfiguration.<Boolean>builder()
+          .key("ALLOW_EXTERNAL_CATALOG_CREDENTIAL_VENDING")
+          .catalogConfig("enable.credential.vending")
```

Comment:
It looks like this name is not specific to external catalogs: `enable.credential.vending`